metricbeat/module/rabbitmq/connection/connection_test.go

```diff
@@ -14,13 +14,21 @@ import (
 )
 
 func TestFetchEventContents(t *testing.T) {
-	absPath, err := filepath.Abs("../_meta/testdata/")
+	absPath, _ := filepath.Abs("../_meta/testdata/")
```

Comment:
I would add a assert.NoError( for these erors. This will make debugging test much easier.